### DISCUSSION TITLE
Is a Xrdp connecton possible via raspap ?

### DISCUSSION THREAD

**Discussion by Striker62** [RaspAP/raspap-webgui#1567] [Q&A]
I'd like to user xrpd acces to my pi via paspap. Is this possible ? and how ?

**Comment by billz** [comment]
RaspAP doesn't support desktop distros (see the FAQ). However, it might be possible to install a desktop environment on RPi OS Lite, then install Xrdp. Might work, might not... you'll be the first to find out. Give it a shot and report back.

**Comment by Striker62** [comment]
I have installed Bullseye 64 on Pi4 with Desktop. Raspap works great. I also can access via xrdp in my local LAN
I think it is probably the port for xrdp that has to be open.
I tried to access the raspberry via SSH that is also not working.

I'm not so experienced with raspap so I can not manga the firewall settings .
It would be great, if you're thinking that it is a Firewall issue, to send me a sample for the firewall setings for ssh. Than I figure out the port for xrdp an add this .

If
